modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java

```diff
@@ -67,11 +68,10 @@ public MutateTask(GAConfiguration config) {
      * @return Gene primary keys
      */
     private List<Long> getMutatedGenes() {
-        List<Long> mutatedGenes = new ArrayList();
+        List<Long> mutatedGenes = new ArrayList<Long>();
         config.getChromosomeLength();
 
         for (int i = 0; i < config.getChromosomeLength(); i++) {
-            // Gene gene=config.getGenePool().get(selectRandomIndex(config.getGenePool().size()));
             mutatedGenes.add(selectGene(i));
         }
 
```

Comment:
Brace style for lines 143 and 146

And for 187: unnecessary local variable?

Reply:
Brace style for lines 143 and 146. Fixed.

And for 187: unnecessary local variable - This variable is necessary.